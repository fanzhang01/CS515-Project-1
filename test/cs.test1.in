Name,Score,Year
Alice,10,2020
Bob,15,2021
Charlie,20,2020