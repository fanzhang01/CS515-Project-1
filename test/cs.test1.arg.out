Sum of 'Score': 45.0
Sum of 'Year': 6061.0
